kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/NMDbusConnectorTest.java

```diff
@@ -1813,7 +1853,7 @@ private void thenModemStatusHasCorrectValues(boolean hasBearers, boolean hasSims
         assertTrue(modemStatus.isGpsSupported());
         assertEquals(EnumSet.of(ModemGpsMode.UNMANAGED, ModemGpsMode.MANAGED_GPS), modemStatus.getSupporteGpsModes());
         assertFalse(modemStatus.isSimLocked());
-        assertEquals(ModemConnectionStatus.REGISTERED, modemStatus.getConnectionStatus());
+        assertEquals(ModemConnectionStatus.FAILED, modemStatus.getConnectionStatus());
```

Comment:
Personally I don't think we should change existing test expectations when we introduce a new feature.

All the old tests still need to pass as before, we just need to test what we added. Is there an easy way to avoid changing existing tests here? If the solution is worse than what is currently implemented ignore this comment.

Reply:
Yes, I agree. However, I chose this solution for two reasons:

1. add the modem status as a parameter requires an huge refactoring of the tests
2. the modem status is not really used in the other tests.

I'd prefer not to change this.